src/monitor.cc

```diff
@@ -614,6 +695,34 @@ bool NodeMonitor::sendReport() {
         snprintf(buffer, sizeof(buffer), "\"health_status_code\":%d,", stats.healthStatusCode_);
         data.append(buffer);
     }
+
+    // append gc stats
+    {
+        GCUsageTracker& tracker = getGCUsageTracker();
+
+        snprintf(buffer, sizeof(buffer), "\"gc\":{" );
+        data.append(buffer);
+
+        for (int i=0; i < GCUsageTracker::kNumGCTypes; ++i) {
+            v8::GCType type = GCUsageTracker::indexTov8GCType(i);
+            const GCStat stat = tracker.GetGCUsage( type )->EndInterval();
+
+            snprintf(buffer, sizeof(buffer), "\"%s\":{", GCUsageTracker::indexToString(i) );
+            data.append(buffer);
+
+            snprintf(buffer, sizeof(buffer), "\"count\":%lu,\"elapsed_ms\":%1.3f,\"max_ms\":%1.3f},",
+                     stat.numCalls, stat.cumulativeTime / (1000 * 1000.0), stat.maxTime / (1000*1000.0) );
```

Comment:
Fair enough -- compiler will optimize this at compile time into the same value, but I'll keep it consistent in the code.